docs/dev/deps.rst

```diff
@@ -153,32 +152,158 @@ detect the other dependencies within the project. The `+snyk-test` Earthly
 target is written to avoid this issue and allow Snyk to accurately detect other
 dependencies within the project.
 
-Due to difficultry coordinating the behavior of Snyk and Silk at time of
+Due to difficulty coordinating the behavior of Snyk and Silk at time of
 writing, vulnerability collection is partially a manual process. This is
 especially viable as the native code contains a very small number of
 dependencies and it is trivial to validate the output of Snyk by hand.
 
 .. seealso:: The `releasing.snyk` step of the release process
 
-.. _snyk: https://app.snyk.io
 
-.. program:: tools/snyk-vulns.py
-.. file:: tools/snyk-vulns.py
+.. _vuln-reporting:
 
-   A Python script that generates the third-party vulnerability report that is
-   included in the release archive. This script is used by the `+vuln-report-md`
-   Earthly target.
+3rd-Party Dependency Vulnerability Reporting
+********************************************
+
+Vulnerabilities in :term:`bundled dependencies <bundled dependency>` are tracked
+by Snyk, but we maintain a hand-written document that details the
+vulnerabilities in current and past dependencies of in-support release versions.
+
+.. file:: etc/third_party_vulnerabilities.md
+
+   The third-party dependency vulnerabily report. This file is stored in the
+   repository and updated manually as vulnerabilities are added/removed.
+
+   .. seealso:: At release-time, this file is added to the release archive. See:
+      `releasing.vuln-report`
+
+
+Updating the Vulnerability Report
+=================================
+
+When updating `etc/third_party_vulnerabilities.md`, perform the following steps:
+
+1. Open the Snyk_ web UI and sign in via SSO.
+2. Open `this Snyk search query`__ (Find the **mongodb/mongo-c-driver** CLI
+   target within the **dev-prod** organization. Do not use the *GitHub target*:
+   That one is not currently useful to us.)
+
+   __ https://app.snyk.io/org/dev-prod/projects?searchQuery=mongo-c-driver&filters[Integrations]=cli
+3. Expand the **mongodb/mongo-c-driver** target, and then expand all **currently
+   supported release versions**. (If you are preparing for a new release, that
+   version should also be available and used after the `releasing.snyk` process
+   has been completed.)
+4. Take note of *all unique vulnerabilities amongst all supported versions'*
+   that are listed in Snyk. These will be the *relevant* vulnerabilities.
```

Comment:
May want to add a note prior to this step to [ignore/deactivate/remove](https://github.com/mongodb/mongo-cxx-driver/blob/dc6da28357136028fc62fb15bec35fde92601e3b/etc/releasing.md#check-and-update-snyk) any unrelated projects Snyk may have detected on its own.

Reply:
I haven't seen this occur in this repo? Running with `--unmanaged` seems to set the scanning mode that we want.

Reply:
Snyk seems to auto-detect any "package"-like files it finds in the repository. If this doesn't affect the C Driver, I think it is fine to leave it to the other Snyk examination steps to handle if it does become an issue.

Reply:
I think it is a non-issue here because we scan the only the `src/` directory (otherwise it fails to pick up Zlib), so those package-like files are not visible.